src/kubectlUtils.ts

```diff
@@ -151,16 +167,18 @@ export async function getPodSelector(resource: string, kubectl: Kubectl): Promis
     });
 }
 
-export async function getPods(kubectl: Kubectl, selector: any): Promise<PodInfo[]> {
-    if (!selector) {
-        return [];
+export async function getPods(kubectl: Kubectl, selector: any, namespace: string = null): Promise<PodInfo[]> {
+    let ns = namespace;
+    if (!ns) {
+        ns = await currentNamespace(kubectl);
+    }

```

Comment:
Would `const ns = namespace || await currentNamespace(kubectl);` do the same thing?  This seems to be the JS idiom for null coalescence.

Reply:
done.